luxonis_train/loaders/luxonis_loader_torch.py

```diff
@@ -119,7 +119,12 @@ def get(self, idx: int) -> tuple[Tensor, Labels]:
 
     @override
     def get_classes(self) -> dict[str, list[str]]:
```

Comment:
We should change the return type to the class mapping instead of extracting just the names so we can have more flexibility